tests/backend/test_states_probabilities.py

```diff
@@ -67,7 +67,7 @@ def test_nongaussian(self, a, phi, setup_backend, cutoff, tol):
             prob_n = state.fock_prob([n, cutoff // 2])
             assert np.allclose(prob_n, ref_probs[n], atol=tol, rtol=0)
 
-
+@pytest.mark.backends("fock", "tf","gaussian")
```

Comment:
```suggestion
@pytest.mark.backends("fock", "tf", "gaussian")
```

is this not supported on the bosonic backend?

Reply:
Not at the moment. The main focus so far for the backend's use so far has been GKP states, where you only need Gaussian measurements. We plan to roll out Fock measurements as a future improvement.